rs.py

```diff
@@ -44,4 +44,4 @@ def classify():
 
 if __name__ == '__main__':
     configure_service('resources/config/app_config.cfg')
```

Comment:
Ok, these would be better suited as environment variables. Also how large are these classifiers?